src/ui/components/navigation/navigationcomponent.js

```diff
@@ -472,8 +473,23 @@ export default class NavigationComponent extends Component {
     }
 
     // We want to persist the params from the existing URL to the new
-    // URLS we create.
+    // URLS we create, except for facets/filters/pagination
     params.set('tabOrder', this._tabOrder);
+
+    params.delete(StorageKeys.SEARCH_OFFSET);
+    params.delete(StorageKeys.FILTER);
+    removeParamsWithPrefixes(
+      params,
+      this.componentManager.getComponentNamesForComponentTypes([
+        'Facets',
```

Comment:
missing Geolocation filter and FilterSearch here. Because the SDK is beautiful those actually use StorageKeys.GEOLOCATION and StorageKeys.FILTER, instead of the name they get in config. On top of that we're slightly uncomfortable refactoring these for url backwards compatibility. hooray for the sdk!

Reply:
OOF. What is the difference between geolocationcomponent.js and locationbiascomponent.js. I've been looking at the former (which also sadly does not use the name they get in the config).

Reply:
geolocationcomponent.js lets you either search in its own searchbar for a location to filter by, or you can click a button and it will use your current location.
![image](https://user-images.githubusercontent.com/23005393/88403356-70d6bb80-cd9a-11ea-9005-ce0fbdf000fa.png)

locationbiascomponent is the location thing you see at the footer of every jambo page. It just says what your current location used in the sdk is, and lets you update it with the "update your location" button. I don't think this affects persistent storage at all but would have to double check
![image](https://user-images.githubusercontent.com/23005393/88403145-20f7f480-cd9a-11ea-98cc-9e42941b94b4.png)

Reply:
Ah nice ok, that's what I thought. On my global search for `StorageKeys.GEOLOCATION`, I only see it set in globalStorage. I don't think it's something that needs to be removed from params for filters, right?

Reply:
yup not needed for filters